test/food.spec.js

```diff
@@ -54,6 +61,14 @@ describe('api', () => {
         })
     })
 
+    test("It can't create a food in the database if missing parameters", () => {
+      return request(app).post('/api/v1/foods')
+        .send('name=Test')
+        .then(response => {
+          expect(response.statusCode).toBe(500)
+        })
```

Comment:
Good start to edge cases, I think I'm going to add one for `DELETE`, and anything else we think of along the way!